src/EventManagement.Services/PaymentMethodService.cs

```diff
@@ -4,40 +4,37 @@
 using losol.EventManagement.Domain;
 using losol.EventManagement.Infrastructure;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
 
 namespace losol.EventManagement.Services
 {
 	public class PaymentMethodService : IPaymentMethodService
 	{
-		private readonly ApplicationDbContext _db;
+        private readonly IEnumerable<PaymentMethod> paymentMethods;
```

Comment:
Could/should we move this to the OrdersService?

Reply:
What makes you think so?